plugins/repository-s3/src/yamlRestTest/resources/rest-api-spec/test/repository_s3/20_repository_permanent_credentials.yml

```diff
@@ -11,6 +11,7 @@ setup:
           type: s3
           settings:
             bucket: ${permanent_bucket}
+            region: ${permanent_region}
```

Comment:
Would we need to add the region in each YAML if we just add it in the gradle file? Eg: https://github.com/opensearch-project/OpenSearch/blob/584617e519226b93cda6eb014a5baadf968e34c2/plugins/repository-s3/build.gradle#L254

If no, adding it in the gradle run configuration might be the better way to do it in case we add new test cases here.

Reply:
I feel like it's more readable to leave it here and keep the configuration together.